jsdependency/jspackage.go

```diff
@@ -47,21 +47,44 @@ func DetectTool(absPackageJSONDir string) Tool {
 }
 
 // Remove removes installed js dependencies using the selected package manager
-func Remove(packageManager Tool, commandScope CommandScope, pkg ...string) error {
+func Remove(packageManager Tool, commandScope CommandScope, pkg ...string) (*command.Model, error) {
```

Comment:
as these commands are not performing any command, may i would call them: `RemoveCommand`